src/components/shared/VolumeChart/index.tsx

```diff
@@ -2,12 +2,9 @@ import { Select, Space } from 'antd'
 import { t, Trans } from '@lingui/macro'
```

Comment:
Random thoughts while testing this (not related to anything in particular, these are existing issues):
- If the user starts at a 30-day window and drops to a 7-day window, presumably the 7 day data is a subset of the 30 day. Could we potentially cache the 30 day data so that we don't need to refetch data for the 7 day window?
- Could show a progressive loading state based on the number of promises waiting to be settled?